inputs/templates/test/SVConcordance/SVConcordance.json.tmpl

```diff
@@ -0,0 +1,19 @@
+{
+  "SVConcordance.gatk_docker":{{ dockers.gatk_docker_concordance | tojson }},
+  "SVConcordance.sv_base_mini_docker": {{ dockers.sv_base_mini_docker | tojson }},
+  "SVConcordance.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
+  "SVConcordance.sv_utils_docker": {{ dockers.sv_utils_docker | tojson }},
+
+  "SVConcordance.eval_vcf" : {{ test_batch.merged_depth_vcf | tojson }},
```

Comment:
I thought the comparison would be the raw calls vs. the clean VCF, not the clustered depth VCF? (If that's the case - the clean VCF should be GATK-ready except for dropping CTX/CPX for your tool, so I'm not sure that the call to SvUtilsFixVcf is necessary)

Reply:
Good catch, fixed